satpy/tests/reader_tests/test_avhrr_l1b_gaclac.py

```diff
@@ -154,6 +155,36 @@ def test_init(self):
                 self.assertIs(fh.reader_class, reader_cls,
                               'Wrong reader class assigned to {}'.format(filename))
 
+    def test_fsfile(self):
+        """Test the use of FSFile filename"""
+        import satpy
+        from satpy.readers import FSFile
+
+        coords = np.arange(4).reshape((2, 2))
+        channels = np.arange(4*6).reshape((2, 2, 6))
+        utcs = np.array(['2000-01-01', '2020-01-01'], dtype='datetime64')
+        reader_mock = mock.MagicMock()
+        reader_mock.get_lonlat.return_value = coords, coords
+        reader_mock.get_calibrated_channels.return_value = channels
+        reader_mock.mask = [True]
+        reader_mock.get_times.return_value = utcs
+        self.pygac.gac_klm.GACKLMReader.return_value = reader_mock
```

Comment:
this test doesn't seem to be using anything from the current testcase class, so how about creating a new testcase for this and putting all these initialisations in the `setUp`?